addons/web/static/src/core/assets.js

```diff
@@ -59,15 +65,26 @@ export const _loadCSS = (assets.loadCSS = memoize(function loadCSS(url) {
     linkEl.type = "text/css";
     linkEl.rel = "stylesheet";
     linkEl.href = url;
-    document.head.appendChild(linkEl);
-    return new Promise(function (resolve, reject) {
+    const promise = new Promise((resolve, reject) => {
         linkEl.addEventListener("load", () => resolve(true));
-        linkEl.addEventListener("error", () => {
-            reject(new AssetsLoadingError(`The loading of ${url} failed`));
+        linkEl.addEventListener("error", async () => {
+            if (retryCount < assets.retries.count) {
+                await new Promise(resolve => setTimeout(resolve, assets.retries.delay + assets.retries.extraDelay * retryCount));
+                linkEl.remove();
+                loadCSS(url, retryCount + 1).then(resolve).catch(reject);
+            } else {
+                reject(new AssetsLoadingError(`The loading of ${url} failed`));
+            }
         });
     });
+    document.head.appendChild(linkEl);
+    return promise;
 }));
 
+export const _setRetries = (assets.setRetries = function setRetries(count = 3, delay = 5000, extraDelay = 2500) {
+    assets.retries = {count: count, delay: delay, extraDelay: extraDelay};
+});
+
```

Comment:
Could you explain why the code is exported twice? Is there an actual reason behind it?

Reply:
No, there was no reason, I now included the whole code in the `setRetries` export.

Reply:
Removed the `setRetries` method altogether, since the `assets` was exported anyway.